plugins/shared/info-panel/index.js

```diff
@@ -295,8 +295,12 @@ class InfoPanel {
                     e.preventDefault();
                     e.stopPropagation();
 
-                    // TODO: This attempts to scroll to fixed elements
-                    $(document).scrollTop(error.$el.offset().top - 80);
+                    // Add tabIndex if element is not currently focusable
+                    if (!error.$el.is(axs.utils.FOCUSABLE_ELEMENTS_SELECTOR)){
```

Comment:
I believe that will be caught in the FOCUS_ELEMENT_SELECTOR. It's value is: 

```input:not([type=hidden]):not([disabled]),select:not([disabled]),textarea:not([disabled]),button:not([disabled]),a[href],iframe,[tabindex]```

Since that [tabindex] is there, would it then cause the if to be `FALSE` and it won't add a tabindex?